.travis.yml

```diff
@@ -20,14 +20,12 @@ env:
   matrix:
   - INSTANCE=chefdk-debian-9
   - INSTANCE=chefdk-debian-10
-  - INSTANCE=chefdk-centos-6
   - INSTANCE=chefdk-centos-7
-  - INSTANCE=chefdk-centos-7 CHEF_VERSION=13
-  - INSTANCE=chefdk-centos-7 CHEF_VERSION=14
-  - INSTANCE=chefdk-ubuntu-1604
+  - INSTANCE=chefdk-centos-8
+  - INSTANCE=chefdk-centos-7 CHEF_VERSION=15
   - INSTANCE=chefdk-ubuntu-1804
-  - INSTANCE=chefdk-ubuntu-1604 CHEF_VERSION=13
```

Comment:
Since we still support 13 we need to figure out how to test on 13

Reply:
added 13 back